Mathlib/Data/Real/Irrational.lean

```diff
@@ -29,6 +29,22 @@ def Irrational (x : ℝ) :=
   x ∉ Set.range ((↑) : ℚ → ℝ)
 #align irrational Irrational
 
+@[simp] theorem not_irrational_zero : ¬Irrational 0 := not_not_intro ⟨0, Rat.cast_zero⟩
+@[simp] theorem not_irrational_one : ¬Irrational 1 := not_not_intro ⟨1, Rat.cast_one⟩
+
+@[simp] theorem not_irrational_natCast (n : ℕ) : ¬Irrational n :=
```

Comment:
See `{Nat|Int|Rat}.not_irrational`.

Reply:
Removed these for now; though I think the names should be changed / deprecated in a later PR